library/src/morpheus-master/sass/modules/tool/_base.scss

```diff
@@ -82,14 +82,22 @@
 		& ~ .#{$namespace}toolTitleNav__button_container > .#{$namespace}directUrl__dropDown{
 			display: none;
 			.dropDown_close{
-				color: #952525;
+				@include sakai_secondary_button();
 				cursor: pointer;
 				float: right;
 				font-size: 1.5em;
-				
 				@media #{$phone}{
 					font-size: 1.4em;
 				}
+				&:active, &:focus, &:hover {
+					&:before {
+						color:var(--sakai-color-red--darker-2);
+					}
+				}
+				&:before {
+					@extend .fa;
+					color:var(--sakai-color-red--darker-2);
+				}
```

Comment:
Normal text colours here are probably okay. This doesn't need to be a red "X" in my opinion.